### ISSUE TITLE
Adapt to current version of react

### ISSUE THREAD

**Issue by AmrAhmedA** [catalinmiron/react-typical#70]
While installing react-typical using the react version 17.0.2, I stumbled upon the following error, it seems that your library needs an older version of react/NPM to function properly. So I need to downgrade my package so it can works fine.
any other solution would be appreciated.
![personal_portfolio](https://user-images.githubusercontent.com/47694180/151347632-89ffe804-4629-4a30-aae5-0e39ca21f844.jpg)

**Comment by Rashed360** [comment]
> While installing react-typical using the react version 17.0.2, I stumbled upon the following error, it seems that your library needs an older version of react/NPM to function properly. So I need to downgrade my package so it can works fine. any other solution would be appreciated. ![personal_portfolio](https://user-images.githubusercontent.com/47694180/151347632-89ffe804-4629-4a30-aae5-0e39ca21f844.jpg)

Did you find any solution?

**Comment by AmrAhmedA** [comment]
> > While installing react-typical using the react version 17.0.2, I stumbled upon the following error, it seems that your library needs an older version of react/NPM to function properly. So I need to downgrade my package so it can works fine. any other solution would be appreciated. ![personal_portfolio](https://user-images.githubusercontent.com/47694180/151347632-89ffe804-4629-4a30-aae5-0e39ca21f844.jpg)
> 
> Did you find any solution?

Yes, I found out that the problem lies in the react-typical package it's self, it's peer dependencies has an older version of react, basically it needs that old version of react to function properly, you can install it via this command
`$ npm i react-typical --legacy-peer-deps`
--legacy-peer-deps tells NPM to ignore peer deps and proceed with the installation. it might cause some unwanted behavior. but I have tested it myself and it works fine with no unwanted behaviors.

**Comment by sridharrajaram** [comment]
> > > While installing react-typical using the react version 17.0.2, I stumbled upon the following error, it seems that your library needs an older version of react/NPM to function properly. So I need to downgrade my package so it can works fine. any other solution would be appreciated. ![personal_portfolio](https://user-images.githubusercontent.com/47694180/151347632-89ffe804-4629-4a30-aae5-0e39ca21f844.jpg)
> > 
> > 
> > Did you find any solution?
> 
> Yes, I found out that the problem lies in the react-typical package it's self, it's peer dependencies has an older version of react, basically it needs that old version of react to function properly, you can install it via this command `$ npm i react-typical --legacy-peer-deps` --legacy-peer-deps tells NPM to ignore peer deps and proceed with the installation. it might cause some unwanted behavior. but I have tested it myself and it works fine with no unwanted behaviors.

Hi, I tried with $npm i react-typical --force. I see sometimes while rendering the timer running fast.
